saleor/graphql/checkout/mutations/utils.py

```diff
@@ -472,6 +473,11 @@ def group_lines_input_data_on_update(
             line_data.custom_price = line["price"]
             line_data.custom_price_to_update = True
 
+        if line_data.metadata_list and metadata_list:
+            line_data.metadata_list += metadata_list
```

Comment:
Codecov says that this if is not tested. From what I see, we have some tests for validating the input, adding a new field, overwriting the field, but this report could be valid, as I don't see a test, where we have a key-1 in meta, and we adding key-2.